comeet/UI/RoomsList/RoomsListViewModel.swift

```diff
@@ -60,6 +60,8 @@ class RoomsListViewModel : BaseViewModel {
     
     func change(date: Date) {
         selectedDate = date
+        startDate = startDate.changeYearMonthDay(newDate: date)
```

Comment:
It would help to have doc comments on the variable declarations in order to clarify different concepts represented by the same data type.